js/common/bootstrap/errors.js

```diff
@@ -79,15 +79,15 @@
     "ERROR_ARANGO_DOCUMENT_NOT_FOUND" : { "code" : 1202, "message" : "document not found" },
     "ERROR_ARANGO_DATA_SOURCE_NOT_FOUND" : { "code" : 1203, "message" : "collection or view not found" },
     "ERROR_ARANGO_COLLECTION_PARAMETER_MISSING" : { "code" : 1204, "message" : "parameter 'collection' not found" },
-    "ERROR_ARANGO_DOCUMENT_HANDLE_BAD" : { "code" : 1205, "message" : "illegal document handle" },
+    "ERROR_ARANGO_DOCUMENT_HANDLE_BAD" : { "code" : 1205, "message" : "illegal document identifier" },
```

Comment:
change is good, but it may break client applications/driver tests filtering on error messages